src/vuestic-theme/vuestic-components/vuestic-grid/_grid-global-styles.scss

```diff
@@ -56,6 +56,7 @@
 
 .va-row {
   @include va-layout();
+  flex-wrap: wrap;
```

Comment:
Please think calmly what you did here.
Check css with sassmeister or something in case you just guessed this line.